src/components/pages/coach/GenerateReport/index.jsx

```diff
@@ -227,13 +228,21 @@ class GenerateReport extends React.Component{
     }
 
     _getReturnValue = (item, condition) => {
+        console.log(item);
```

Comment:
Are these console logs needed? I think you should comment them out at least.

Reply:
Yes, for realtime runing-system debugging.